### PR TITLE
re-implement word counting to work with unicode letters

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDESK-5352

### CODE REVIEW

**Comment by ride90** [comment]
@tomaskikutis check failing tests and linter

File: tests/io/feed_parsers/pa_nitf_test.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def test_slugline(self):
         self.assertEqual(self.item.get('format'), 'HTML')
         self.assertEqual(4, len(self.item.get('subject')))
         self.assertIn('Trivia (Oct 14)', self.item.get('keywords'))
-        self.assertEqual(637, self.item.get('word_count'))
+        self.assertTrue(type(self.item.get('word_count')) is int)
```
Comment (petrjasek) [inline tests/io/feed_parsers/pa_nitf_test.py:57]: you can do `self.assertIsInstance(self.item.get('word_count'), int)`

File: tests/io/feed_parsers/nitf_tests.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def test_anpa_category(self):
         self.assertEqual(self.item.get('anpa_category')[0]['qcode'], 'a')
 
     def test_word_count(self):
-        self.assertEqual(349, self.item.get('word_count'))
+        self.assertIsInstance(self.item.get('word_count'), int)
```
Comment (ride90) [inline tests/io/feed_parsers/nitf_tests.py:91]: Checking the type is ok, but we also must ensure the count equality.

Comment (tomaskikutis) [reply tests/io/feed_parsers/nitf_tests.py:91]: The question is what is the correct number? It's not practical to go over hundreds of words to count it manually in order to be able to compare the results. There are also various ways to count formatting. Even if you counted words manually it's likely that you would get a different result than it was specified in the assert. That's why I prefer to stick to specific scenarios in `text_utils.py`.

Comment (ride90) [reply tests/io/feed_parsers/nitf_tests.py:91]: In this case what all these test_word_count tests do?

Comment (ride90) [reply tests/io/feed_parsers/nitf_tests.py:91]: ok, let's keep it as it is

Comment (tomaskikutis) [reply tests/io/feed_parsers/nitf_tests.py:91]: > In this case what all these test_word_count tests do?

I would remove them, but kept just to check that there's some number populated. Not sure if it's very useful.

**Pull request merged**